src/modules/schema.js

```diff
@@ -0,0 +1,240 @@
+import { randomAsHex, encodeAddress, decodeAddress } from '@polkadot/util-crypto';
+import { u8aToU8a, u8aToString, u8aToHex } from '@polkadot/util';
+import { validate } from 'jsonschema';
+import axios from 'axios';
+
+import { getHexIdentifierFromDID } from '../utils/did';
+import { getSignatureFromKeyringPair } from '../utils/misc';
+import { isHexWithGivenByteSize } from '../utils/codec';
+import Signature from '../signatures/signature';
+
+import { DockBlobIdByteSize } from './blob';
+
+// Supported schemas
+import JSONSchema07 from '../utils/vc/schemas/schema-draft-07';
+
+// Blob qualifier
+export const BlobQualifier = 'blob:dock:';
+
+// Size of the blob id in bytes
+export const BLOB_ID_MAX_BYTE_SIZE = 32;
+
+// Maximum size of the blob in bytes
+// implementer may choose to implement this as a dynamic config option settable with the `parameter_type!` macro
+export const BLOB_MAX_BYTE_SIZE = 1024;
+
+/**
+ * Check if the given identifier is 32 byte hex
+ * @param {string} identifier - The identifier to check.
+ * @return {void} Throws exception if invalid identifier
+ */
+export function validateBlobDIDHexIdentifier(identifier) {
```

Comment:
The function name has DID but its not about a DID. And the logic is same as `validateDockDIDHexIdentifier`. Abstract the common part in a separate function to accept identifier and qualifier.